docs/monthly-meeting/2023-05.md

```diff
@@ -0,0 +1,121 @@
+- Joanna, Python education
+- Jim, from Vancouver, Canada; reads docs & fixes things he notices
+- Manuel Kaufmann from Argentina now in Spain, translations, Read the Docs, CPython translation in Spanish, PSF Fellow
+- Marcus Sherman, from Portland, Maine @ Northeastern University, `@betteridiot`, biologist by training
+- Ned, from Boston, made the docs official, eager to see this group grow into a large & welcoming community
+- Cristián Maureira-Fredes, CPython Spanish translation
+- Keto Zhang, software engineer in astronomy, went to Cam's talk at PyCon
+- CAM, gave the talk at PyCon, new Python core dev, from Huntsville, Alabama, Spyder maintainer, looking to be a full-time open-source dev & documentarian
+- Ezio, core Dev, working on Docs and infrastucture
+
+
+## Reports and celebrations
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+Congrats to CAM!
+
+Congrats Ned for the [keynote at PyCon US 2023](https://us.pycon.org/2023/about/keynote-speakers/#ned-batchelder)
+
+
+* [PyCon](https://us.pycon.org/2023/), April 19-27 2023:
+  * CAM & Erlend's talk, "[Iteration toward Transformation of the Python Documentation](https://us.pycon.org/2023/schedule/presentation/88/)", went well, good discussions afterwards and many joined the Discord wanting to contribute
+  * Sprints: some docs contributors, including people making their very first CPython contribution
+  * Sprints: [Hugo] switched CPython docs previews from Netlify to RtD.
+    * Main benefit: multiple admins instead of single-person bottleneck with Netlify.
+    * https://github.com/python/cpython/pull/103843
+* GH action to update the link from the PR - we don't need one for Netlify
+    * [JDLH thinks this means] GitHub action to update a link to a documentation preview on ReadTheDocs from the Github Pull Request page. We don't need such a preview for the current docs as published at Netlify.
+    * Example of preview link: Pull Request [#104013, **gh-104010: Separate and improve docs for typing.get_origin and typing.get_args**](https://github.com/python/cpython/pull/104013), contains a link to a preview of the documentation reflecting the Pull Request's effects, at https://cpython-previews--104013.org.readthedocs.build/en/104013/ .
+* For moving main docs, a few more things are needed, but they're tracked at the RtD side
+* We're using a personal account (Mariatta's). Not ideal. Right now community version of RtD doesn't support multiple owners, there are workarounds. Organizations are only in the commercial version so far.
+* would be nice to have an ORG account.
```

Comment:
Read the Docs Community does not support organizations yet. However, it _does support_ multiple maintainers with admin permissions for the same project. As a example, this is how we have configured our own documentation:




![Screenshot_2023-06-05_20-51-07](https://github.com/python/docs-community/assets/244656/ba291b8f-5040-445c-a57c-502bdf79fbaa)

Reply:
Right, thanks, and I believe we use that on our main RTD-hosted/previewed stuff, with at least three-ish people added there for each. If I understand the concern here, though, it is that each project is still technically "owned" by one individual person's personal account, leading to an eventual single point of failure, alongside the inability to centrally manage many different projects with different owners/maintainers owned by one FOSS entity, its all scattered between individual people's accounts.

This because particularly problematic if we end up moving the hosting of the Python docs to RTD, since it means the ultimate responsibility for a docs site that gets tens of millions of hits per month would have to rest on the shoulders of one person.

Reply:
Sure, it would be nice for central management and an org account.

But practically, I'm not too worried about that. The main thing is to make sure enough have admin access to keep the bus factor healthy.

Currently it's just Mariatta and me for the CPython previews. Would any other core devs like access too?

Reply:
Not a core dev, but for consistency purposes you could add Ee. And if you need an extra person and don't have someone better, you could add me if needed—I'm not actually an admin of any other RTD-hosted project right now, so it would be helpful to have at least one I can get familiar with.

Reply:
@CAM-Gerlach Please create an RtD account, lete know the username, and I'll add you!

Reply:
@hugovk Actually seems like I do have a RTD account, under the username `CAM-Gerlach`, in case you want to add that.

Reply:
`CAM-Gerlach` invited!